bedrock/firefox/views.py

```diff
@@ -322,6 +322,15 @@ def show_57_firstrun(version):
     return version >= Version('57.0')
 
 
+def show_rocket_whatsnew(version):
```

Comment:
Would this mean anyone updating from < 57.0 -> 57.0.1 misses the Chris Beard letter? I don't see any specific ask in the bug to target dot releases for 57 specifically, although it is a bit vague on info. There is a [bug for 58](https://bugzilla.mozilla.org/show_bug.cgi?id=1428419) though?

Also, this logic is targeting the `id` locale only. Should we be showing the mobile `/whatsnew` page for all locales on dot releases instead?  As this stands, every other locale would see the cbeard letter again. If we changed the logic to show the mobile `/whatsnew` page for all locales, we could then use a locale-specific template for this page variation?

Some tests for whatever we end up with here would be good too.

Reply:
Could we maybe do some logic like:

- Users updating from < 57.0 see cbeard letter
- Users updating from > 57.0 see mobile promo

This is the same kind of logic we did for the 56 whatsnew page. As it stands here, people would see the cbeard letter multiple times, unless they have `id` as their preferred locale?